doc/modules/permutation_importance.rst

```diff
@@ -6,15 +6,27 @@ Permutation feature importance
 
 .. currentmodule:: sklearn.inspection
 
-Permutation feature importance is a model inspection technique that can be used
-for any :term:`fitted` :term:`estimator` when the data is tabular. This is
-especially useful for non-linear or opaque :term:`estimators`. The permutation
-feature importance is defined to be the decrease in a model score when a single
-feature value is randomly shuffled [1]_. This procedure breaks the relationship
-between the feature and the target, thus the drop in the model score is
-indicative of how much the model depends on the feature. This technique
-benefits from being model agnostic and can be calculated many times with
-different permutations of the feature.
+Permutation feature importance is a model inspection technique that measures the
+contribution of each feature to a :term:`fitted` model's performance on a given
+tabular dataset. This technique is particularly useful for non-linear or opaque
+:term:`estimators`, and involves randomly shuffling the values of a single
+feature and observing the resulting degradation of the model's score [1]_. By
+breaking the relationship between the feature and the target, we can determine
+how much the model relies on such feature. One key advantage of permutation
+feature importance is that it is model-agnostic, i.e. it can be applied to any
+fitted estimator. Moreover, it can be calculated multiple times with different
+permutations of the feature, providing a robust measure of feature importance
+for the specific trained model.
+
+The figure below shows the permutation feature importance of a
+:class:`~sklearn.ensemble.RandomForestClassifier` trained on an augmented
+version of the titanic dataset that contains a `random_cat` and a `random_num`
```

Comment:
At this stage, the reader does not know what `random_cat` and `random_num` are. I think that we can mention that we included two random features in the dataset and refer to the feature names then.